cuegui/cuegui/MenuActions.py

```diff
@@ -1136,10 +1136,8 @@ def editSize(self, rpcObjects=None):
             body = "Please enter the new subscription size value:\nThis " \
                    "should only be changed by administrators.\nPlease " \
                    "contact the resource department."
-            (value, choice) = QtWidgets.QInputDialog.getDouble(self._caller,
-                                                               title, body,
-                                                               current,
-                                                               0, 50000, 0)
+            (value, choice) = QtWidgets.QInputDialog.getDouble(self._caller, title, body, current,
+                                                               0, 2147483647, 0)
```

Comment:
Might want to move this numerical value to a constant so it reads easier. Could do the same for the other numbers that get passed here too.